front/app/components/EventPreviews/index.tsx

```diff
@@ -54,10 +56,30 @@ const EventPreviews = ({ projectId }: EventPreviewsProps) => {
         >
           {formatMessage(messages.eventPreviewTimelineTitle)}
         </Title>
-        <HorizontalScroll containerRole="listbox" asList>
-          {events.data.map((event) => (
-            <EventPreviewCard key={event.id} event={event} />
-          ))}
+        <HorizontalScroll>
+          <ul style={{ display: 'flex', listStyleType: 'none' }}>
```

Comment:
Let's make the padding zero to maintain the look in master

```suggestion
          <ul style={{ display: 'flex', listStyleType: 'none', padding: '0px' }}>
```